glib-macros/src/lib.rs

```diff
@@ -872,6 +872,7 @@ pub fn cstr_bytes(item: TokenStream) -> TokenStream {
 /// `PropertySet` and `PropertySetNested` if possible.
 ///
 /// The type `Option<T>` is supported as a property only if `Option<T>` implements `ToValueOptional`.
+/// Optional types also require the `nullable` attribute.
```

Comment:
This probably needs a bit more explanation in the docs. It also works without but explaining what the difference between with/without is would be useful here.